### PR TITLE
Develop front admin

### PR DESCRIPTION
**Pull request by Songsterr99**
admin part is added

### CODE REVIEW

**Comment by Songsterr99** [comment]
fixed

File: front/src/components/AdminLabs/AdminLabs.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import './AdminLabs.scss';
+
+
+class AdminLabs extends Component {
+  constructor() {
+    super();
+
+    const data = require('../../data');
```
Comment (AndreiShenets) [inline front/src/components/AdminLabs/AdminLabs.js]: Why require is used instead of import?

File: front/src/components/AdminLabs/AdminLabs.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
```
Comment (AndreiShenets) [inline front/src/components/AdminLabs/AdminLabs.js]: Unnecessary empty line

File: front/src/components/AdminLabs/AdminLabs.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+
+    table{
+      border: 1px solid #ccc;
+      border-collapse: collapse;
+      margin: 0;
+      padding: 0;
+      width: 100%;
+      table-layout: fixed;
+
```
Comment (AndreiShenets) [inline front/src/components/AdminLabs/AdminLabs.scss]: Unnecessary empty line?

File: front/src/components/AdminLabs/AdminLabs.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    h2{
+      font-weight: bold;
+    }
+  }
+
+  .labs-list__table{
+    max-width: $max-width;
+    margin: auto;
+
```
Comment (AndreiShenets) [inline front/src/components/AdminLabs/AdminLabs.scss]: Unnecessary empty line?

File: front/src/components/Footer/Footer.js [hunk 5]
```diff
@@ -1,34 +1,47 @@
...
+    this.lang = this.lang.bind(this);
+  }
+
+  lang(lang) {
+    const i18n = this.props.i18n;
+    i18n.changeLanguage(lang);
+  }
+
   render() {
+    const t = this.props.t;
```
Comment (AndreiShenets) [inline front/src/components/Footer/Footer.js]: It should be done authomatically

File: front/src/components/Header/Header.js [hunk 6]
```diff
@@ -1,28 +1,50 @@
...
+        <Link to={'/users'}>{t('lbUsers')}</Link>
+        <button onClick={logOut}>{t('btnLogout')}</button>
+      </div>
+    ) : (
+      <button className="head__item_right_hide_login" onClick={showLogin}>{t('btnLogin')}</button>
+    );
+    const adminPartResponsive = isAdmin() ? (
+      <ul>
+        <li>Daniil</li>
+        <li> {t('lbCourses')}</li>
```
Comment (AndreiShenets) [inline front/src/components/Header/Header.js]: Unnecessary space after <li>

File: front/src/components/Header/Header.scss [hunk 7]
```diff
@@ -47,6 +39,40 @@ $show-menu: 850px;
     }
 
     .head__item_right_hide {
+      .head__item_right_hide_login{
```
Comment (AndreiShenets) [inline front/src/components/Header/Header.scss]: Shoul be space between style and {. The same for other places

File: front/src/components/Schedule/Schedule.js [hunk 8]
```diff
@@ -1,223 +1,61 @@
...
+    const materials = this.state.materials.map(material =>
+      <li>{material}</li>);
     return (
       <section className="schedule">
         <div className="schedule__list">
           <div className="schedule__list_header">
 
             <div className="schedule__list_header_img">
-              <img src={"/img/courses/" + lab.image} alt=""/>
+              <img src={`/img/courses/${lab.image}`} alt=""/>
```
Comment (AndreiShenets) [inline front/src/components/Schedule/Schedule.js]: Need to discuss image construction

File: front/src/languages/lang.json [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+      "lbLanguage":"Язык",
+      "btnAddCourse":"Добавить курс",
+      "btnLogout":"Выйти",
+      "phSearchCourse":"Искать курс",
+      "lbCourses":"Курсы",
+      "lbUsers":"Пользователи",
+      "phSearchUser":"Искать пользователя",
+      "btnAddUser":"Добавить",
+      "lbUsersHeader":"Пользователи"
+
```
Comment (AndreiShenets) [inline front/src/languages/lang.json]: Unnecessary empty lines

**Pull request merged**